tests/dags/test_dags.py

```diff
@@ -0,0 +1,39 @@
+import glob
+import pytest
+from os import path
+from airflow import models
+from airflow.utils.dag_cycle_tester import test_cycle as cycle_test
```

Comment:
I noticed that in the current `main` branch for Airflow, they have renamed `test_cycle` to [check_cycle](https://github.com/apache/airflow/blob/10421c693199eeea2c1ea54844319080fd6f7153/airflow/utils/dag_cycle_tester.py#L47) but isn't part of the latest Airflow 2.1.4 release. Maybe rename this to `check_cycle`?